aries-site/src/data/structures/templates.js

```diff
@@ -598,3 +598,128 @@ export const templates = [
     tags: ['content', 'layout', 'page layouts', 'responsiveness', 'responsive'],
   },
 ];
+
+// these pages do not necessarily have their own routes but instead
+// link to sections of existing pages
+export const contentLayouts = [
+  {
+    name: 'Single Column - Wizard',
+    category: 'Single Column',
+    description: '',
```

Comment:
Can we omit `description`? Results in an unnecessary / empty `<span />` in the DOM.

Probably need a small update to ContentCard to conditionally render description only if defined.

Reply:
Yes, will update the proptypes on ContentCard to make description optional.